php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2707,10 +2707,15 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     public function handle(ServerRequestInterface $request) : ResponseInterface
     {
-        if ($request->getMethod() === "POST") {
+        if ($request->getMethod() === "POST" && !isset($_POST['ClearInstrument'])) {
```

Comment:
values being checked should come from the `$request` ServerRequestInterface argument to the function, not the `$_POST` superglobal.